src/openvas.c

```diff
@@ -560,6 +560,10 @@ openvas (int argc, char *argv[], char *env[])
     }
   tzset ();
 
+  if (scan_id)
```

Comment:
To make deployment of GOS less error prone and since we currently don't have semantic version as of yet maybe it makes sense to add a
```
#define LOG_REFERENCES_AVAILABLE
```

in gvm libs to verify here if the functions are indeed available.

Otherwise we would require that gvm-libs are updated first before updating openvas. Since we currently don't have semantic versioning this could be messy otherwise.